_health-care/_js/components/personal-information/NameAndGeneralInfoSection.jsx

```diff
@@ -26,8 +26,7 @@ class NameAndGeneralInfoSection extends React.Component {
                   onUserInput={(update) => {this.props.onStateChange('mothersMaidenName', update);}}/>
               <SocialSecurityNumber ssn={this.props.data.socialSecurityNumber}
                   onValueChange={(update) => {this.props.onStateChange('socialSecurityNumber', update);}}/>
-              <Gender gender={this.props.data.gender}
-                  onUserInput={(update) => {this.props.onStateChange('gender', update);}}/>
+              <Gender value={this.props.data.gender}/>
```

Comment:
I don't think you want to remove the prop `onUserInput`. Your function `handleChange` in `Gender.jsx` is referring to it when it does: `this.props.onUserInput(this.refs.gender.value)`. What that's doing is finding a prop on `Gender` (this refers to `Gender`) called `onUserInput` and passing it `this.refs.gender.value`. The line you removed takes `this.refs.gender.value` as `update` and then passes that to another prop on `NameAndGeneralInfoSection` called `onStateChange`. It's a bit confusing to trace how the data is getting passed around, but basically we're working our way up each layer.